test/test_preprocessing_encoders.py

```diff
@@ -211,22 +211,27 @@ def test_robust_ordinal_encoding_categories():
 
 
 def test_robust_ordinal_encoding_transform():
-    encoder = RobustOrdinalEncoder()
-    encoder.fit(ordinal_data)
-    test_data = np.concatenate([ordinal_data, np.array([["waffle", 1213, None]])], axis=0)
-    encoded = encoder.transform(test_data)
-    assert all(list((encoded[:-1] < 3).reshape((-1,))))
-    assert all(list(encoded[-1] == 3))
+    for unknown_as_nan in [True, False]:
```

Comment:
I'd actually prefer if you could create a new test for the specific use case rather than modifying the existing tests. But if you are against that, I suggest using `@pytest.mark.parametrize` instead of this for loop in the tests

Reply:
I tested it that way to avoid code duplication. Changed the code to use `@pytest.mark.parametrize`